### ISSUE TITLE
Write a Python program to check even or odd number

### ISSUE THREAD

**Issue by harshraj8843** [codinasion/program#1390]
### Description

Write a Python program to check even or odd number

```
Input  : 3
Output : Odd Number
```
<details>
<summary>How to contribute</summary>

- Save the solution in `check-even-or-odd-number.py` file
- Add `check-even-or-odd-number.py` file in `check-even-or-odd-number` folder

</details>

**Comment by grraghav120** [comment]
!assign

**Comment by grraghav120** [comment]
Kindly add label hacktoberfest-accepted @harshraj8843 @victoriacheng15

**Comment by victoriacheng15** [comment]
> Kindly add label hacktoberfest-accepted @harshraj8843 @victoriacheng15

The bot will automatically add the label once it is merged

**Comment by harshraj8843** [comment]
Hey @grraghav120 

Sorry, but a different user raised a PR before your assignment

Please contribute to another issue
